docs/output.md

```diff
@@ -600,7 +596,7 @@ Sex.DetERRmine calculates the coverage of your mapped reads on the X and Y chrom
 
 When a bedfile of specific sites is provided, Sex.DetERRmine additionally calculates error bars around each relative coverage estimate. For this estimate to be trustworthy, the sites included in the bedfile should be spaced apart enough that a single sequencing read cannot overlap multiple sites. Hence, when a bedfile has not been provided, this error should be ignored. When a suitable bedfile is provided, each observation of a covered site is independent, and the error around the coverage is equal to the binomial error estimate. This error is then propagated during the calculation of relative coverage for the X and Y chromosomes.
 
-> Note that in nf-core/eager this will be run on single- and double-stranded variants of the same library _separately_. This can also help assess for differential contamination between libraries.
+> Note that in nf-core/eager this will be run on single* and double-stranded variants of the same library _separately_. This can also help assess for differential contamination between libraries.
```

Comment:
```suggestion
> Note that in nf-core/eager this will be run on single- and double-stranded variants of the same library _separately_. This can also help assess for differential contamination between libraries.
```